src/coprocessor/endpoint.rs

```diff
@@ -203,16 +204,19 @@ impl<E: Engine> Endpoint<E> {
                         !req_ctx.context.get_not_fill_cache(),
                         req_ctx.bypass_locks.clone(),
                     );
-                    dag::build_handler(
+                    dag::DagHandlerBuilder::new(
                         dag,
                         ranges,
                         store,
-                        data_version,
                         req_ctx.deadline,
                         batch_row_limit,
                         is_streaming,
```

Comment:
Seems `is_streaming` is another option for `DagHandlerBuilder`, could we specify a builder for it?

Reply:
I expose all compulsary options only to the `new` function. Currently it is not uncessary to provide a modifier for these options.